### PR TITLE
Change file deploy

### PR DESCRIPTION
**Pull request by Any97Cris**

### CODE REVIEW

File: .github/workflows/composer-validate.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+name: Composer Validate
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
```
Comment (vitormattos) [inline .github/workflows/composer-validate.yml]: Could be only the follow to don't fix running the validation only when is a PR to main branch:
```suggestion
on: pull_request
```

File: .github/workflows/composer-validate.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
+
```
Comment (vitormattos) [inline .github/workflows/composer-validate.yml]: Maybe unnecessary. Sounds that this file was generated from a template to create an Action.

```suggestion
```

File: .github/workflows/composer-validate.yml (hunk 1 above)
Comment (giovannism20) [reply .github/workflows/composer-validate.yml]: And we need to guarantee that no one will commit on main branch

**Review by giovannism20 — APPROVED** [review]

File: .github/workflows/composer-validate.yml (hunk 1 above)
Comment (vitormattos) [reply .github/workflows/composer-validate.yml]: This is made as setting at repository.

File: .github/workflows/composer-validate.yml [hunk 3]
```diff
@@ -10,7 +8,7 @@ on:
   pull_request:
     branches: [ main ]
   workflow_dispatch:
-
+on: pull_request
```
Comment (vitormattos) [inline .github/workflows/composer-validate.yml:3]: Acho que vale remover o `on` que está antes deste

**Review by vitormattos — APPROVED** [review]

**Pull request merged**